codegenerator/cli/templates/static/factory_indexer_template/typescript/src/EventHandlers.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Please refer to https://docs.envio.dev for a thorough guide on all Envio indexer features
+ */
+import { UniswapV3Factory, UniswapV3Pool } from "generated";
+
+UniswapV3Factory.PoolCreated.handler(async ({ event, context }) => {
+    const entity = {
+        id: `${event.chainId}_${event.params.pool}`,
+        token0: event.params.token0,
+        token1: event.params.token1,
+        fee: event.params.fee,
+        tickSpacing: event.params.tickSpacing,
+        pool: event.params.pool,
+    };
+
+    context.UniswapV3Factory_PoolCreated.set(entity);
+});
```

Comment:
Let's remove the handler and UniswapV3Factory_PoolCreated since it's not relevant for the template.